scss/_custom-forms.scss

```diff
@@ -64,18 +63,14 @@
   top: .0625rem;
   left: 0;
   display: block;
-  width: 1rem;
-  height: 1rem;
-  font-size: 65%;
```

Comment:
Yup. The font and other type related properties do nothing since we're using `background-image`s for the icon.